polkadot/xcm/xcm-fee-payment-runtime-api/README.md

```diff
@@ -0,0 +1 @@
+License: Apache-2.0
```

Comment:
I think we could just drop this "empty" README.md file

all source files are already specifying GPL3 licence